docs/dapps/oev-rewards/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Getting paid
+pageHeader: dApps → OEV Rewards
+outline: deep
+---
+
+<PageHeader/>
+
+# Getting paid
+
+dApps that use traditional data feeds are constantly exploited by MEV bots that manipulate the order operations around individual data feed updates.
+In practice, this causes dApps to suffer significant and continuous financial losses.
+As the antidote, API3 auctions off the privilege to determine the order of operations around data feed updates to OEV searchers, and pays the auction proceeds to the dApp in the form of OEV Rewards.
+Assuming competitive auctions, OEV Rewards will be equal to the amount that would otherwise have been lost to the MEV bots.
+
+API3 data feeds work identically to traditional data feeds, which means that you do not need to modify your contracts in any way to use them.
+You can drop in API3 data feeds in place of your current data feeds, and immediately start earning OEV Rewards.
+
+## How to get onboard
+
+1. Get assigned a [dApp ID.](/dapps/oev-rewards/dapp-id)
+2. [Deploy](/dapps/integration/contract-integration#deployment-parameters) Api3ReaderProxyV1 contracts with this dApp ID.
+3. [Integrate](/dapps/integration/contract-integration) the Api3ReaderProxyV1 contracts that you have deployed.
+
+At the end of each month, we will send you a report and 80% of the auction proceeds in the native currency of the network that your dApp is deployed on, and keep the rest as the protocol fee.
+
+::: info 💡 Tip
+
+Traditionally, the relationship between dApps and oracles has been a zero-sum game.
+Oracles lured dApps with free services, and later used vendor lock-in to extract as much as possible from them.
+
+OEV Rewards enable dApps and API3 to earn at the same time, which begs the question, at whose expense?
+The answer, in short, is _the MEV mafia_.
```

Comment:
Meh

Reply:
Is your issue only with the style? I think if we tell people that they will receive a service and be paid on top, it's natural for them to think that there's a catch. It's worth telling that we're able to do this by hurting other (bad) people.

Reply:
It was a style comment- absolutely agree on substance- and I realize people will react differently upon reading "XYZ mafia" so either way.

Reply:
I recalled "MEV mafia" being a common saying but after googling it I realize that it's a 3 year-old Chainlink talking point from when they were trying to make fair sequencing happen :sweat_smile: 
Tried again in https://github.com/api3dao/oev-docs/pull/129/commits/3e918381eab8df365ba6d354dc1cf97837be5ec1

Reply:
🤣  . Updated language sounds good